.github/workflows/DEV_CD_backend.yml

```diff
@@ -0,0 +1,45 @@
+name: DEV - Build and Deploy Application to Azure
+
+on:
+  push:
+    branches:
+      - [dev]
+    paths:
+      - "GiEnJul/**"
+  workflow_dispatch:
+
+env:
+  DOTNET_VERSION: "3.1.x"
+  PACKAGE_PATH: "Deploy"
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Github Action
+        uses: actions/checkout@dev
```

Comment:
Based on https://github.com/marketplace/actions/checkout
I think this should be @v2 and then we can specify branch by adding 
`with:
    ref: my-branch`